spec/sauce_helper.rb

```diff
@@ -8,6 +8,39 @@
 
 tunnel_id = "restricted-input"
 
+PLATFORM = ENV["PLATFORM"]
+
+def select_browsers
+  browsers = []
+
+  if !PLATFORM || PLATFORM == "desktop"
+    browsers += [
+      ["Windows 10", "chrome", nil],
+      # Firefox 48 (latest on Sauce) is failing all tests
```

Comment:
I will update it after this gets merged in (unless there are actually code change that need to be made) since the build takes sooooooooo long to run.